### ISSUE TITLE
Privilege Groups

### ISSUE THREAD

**Issue by nikkelj** [ellmetha/django-machina#82]
Eh, I lied, I have a couple more to post ;).

Another administrative feature I've found myself wishing for is a group forum privilege sets model/admin.  Maybe there are some tricks I am missing, but it is quite time consuming to assign privs one by one.

A related issue is that the group forum privileges admin does not by default show the forum each created privilege belongs to, which makes it very hard to manage.  I've tried overloading the forum_privileges package to attempt fixing that, I haven't been able to do that successfully for reasons I don't understand yet (I have overloaded some of the other ones successfully).

**Comment by ellmetha** [comment]
Privileges? There are no "privileges" in django-machina but a permission system (and there is no `forum_privileges` package). You can easily apply common permissions to all your users by creating groups (with all your users in it) and assigning global forum permissions to it. But I guess that you're talking about *roles* (ie. groups of permissions) that could ease the way permissions are assigned or not.

**Comment by nikkelj** [comment]
Yes, I meant apps/forum_permissions.  It appears you guessed correctly, roles would be the correct word to describe my desire.  I desire to assign multiple group permissions to a role once, and then only have to assign roles to boards.

**Comment by nikkelj** [comment]
Globals work in some cases, but I (and I'm sure others) have some use cases that globals don't serve well

**Comment by BrendaH** [comment]
This thread looks quite old, but I found myself wanting the same thing.
I'd like to have a set of permissions ( a 'role' or 'cluster') that I can apply to multiple groups for the same or different forums.
For example "can only read and download" where all permissions are turned off except seeing/reading the forum and downloading attachments. Or one "settings for private forum" that I can apply to specific subforum/group combinations.

What is the view on such a grouping of permissions?

The displaying of the forum in the changelist was recently added by PR #121, by the way.
